CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 3.0.3
+
+- Added documented on `BaseAsynchronousOperation`
```

Comment:
Nitpicking as this is just a change log but should this be "added documentation"?